src/core/swa-cli-persistence-plugin/persistence-cache-plugin.ts

```diff
@@ -30,7 +31,7 @@ export class SWACLIPersistenceCachePlugin implements ICachePlugin {
   public async beforeCacheAccess(cacheContext: TokenCacheContext): Promise<void> {
     logger.silly(`Executing before cache access plugin`);
 
-    const machineId = await getMachineId();
+    const machineId = await getMachineId("shake256", MACHINE_ID_LENGTH);
```

Comment:
any reasons for using different algo for machineId here and what we are logging ? should we use same machineId ?